tests/code/test_minor.py

```diff
@@ -35,44 +43,39 @@ def test_getCourseInformation():
 
 
 @pytest.mark.integration
-def test_updateMinorInterest():
+def test_toggleMinorInterest():
     with mainDB.atomic() as transaction:
-        test_user = User.create(
-            username="FINN",
-            firstName="Not",
-            lastName="Yet",
-            email=f"FINN@berea.edu",
-            bnumber="B91111111")
+        User.create(username="FINN",
+                    firstName="Not",
+                    lastName="Yet",
+                    email=f"FINN@berea.edu",
+                    bnumber="B91111111")
         
         user = User.get_by_id("FINN")
         # make sure users have the default values of false and not interested, respectively
         assert user.minorInterest == False
-        assert user.minorStatus == "No interest"
-        updateMinorInterest("FINN")
+        toggleMinorInterest("FINN")
         
         user = User.get_by_id("FINN")
-        # make sure updateMinorInterest works correctly
+        # make sure toggleMinorInterest works correctly
         assert user.minorInterest == True
-        assert user.minorStatus == "Interested"
         
         # verify unchecking box will restore defaults
-        updateMinorInterest("FINN")
+        toggleMinorInterest("FINN")
         
         user = User.get_by_id("FINN")  
         assert user.minorInterest == False
-        assert user.minorStatus == "No interest"
         transaction.rollback()
 
 @pytest.mark.integration
 def test_getProgramEngagementHistory():
     with mainDB.atomic() as transaction:
         # create test objects
-        test_user = User.create(
-            username="FINN",
-            firstName="Not",
-            lastName="Yet",
-            email=f"FINN@berea.edu",
-            bnumber="B91111111")
+        testUser = User.create(username="FINN",
+                                firstName="Not",
+                                lastName="Yet",
+                                email=f"FINN@berea.edu",
```

Comment:
Nit picky but no need for this to be an f string if we aren't doing any formatting.